packages/SketchMorph2-Core.package/M2CustomizableColorMorph.class/instance/helpText.st

```diff
@@ -0,0 +1,3 @@
+accessing
+helpText
+	^ 'Left click: Add color' , Character cr , 'Right click: Clear'
```

Comment:
"Add current color" instead of just "Add color" would be nicer.

Reply:
We should also mention that left clicking would select the color if one is set already.